.github/workflows/spec-update.yaml

```diff
@@ -69,12 +69,12 @@ jobs:
         run: |
           git config --global user.email "cloudsdk@sap.com"
           git config --global user.name "SAP Cloud SDK Bot"
-          git fetch --unshallow origin main
+          git fetch --unshallow origin v1-main
```

Comment:
[req] We should only do spec update on main. Orchestration will also use this workflow to use the generator. And they should use our main branch.